apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -121,6 +123,28 @@ const CustomizableCurriculumCatalogCard = ({
 }) => {
   const [isAssignDialogOpen, setIsAssignDialogOpen] = useState(false);
 
+  const handleClickLearnMore = () => {
+    analyticsReporter.sendEvent(
+      EVENTS.CURRICULUM_CATALOG_LEARN_MORE_CLICKED_EVENT,
+      {
+        curriculum_offering: courseDisplayNameWithYear,
+      }
+    );
+    window.location.href = pathToCourse;
```

Comment:
Will this open in a new tab if the user wants it to? (Thinking cmd+click, cmd+enter, or right-click+open in new tab)

Reply:
(Same as above comment)